septentrion/cli.py

```diff
@@ -180,6 +197,11 @@ def split_envvar_value(self, rv: str):
     default=configuration.DEFAULTS["ignore_symlinks"],
     help="Ignore migration files that are symlinks",
 )
+@click.option(
+    "--create_table",
```

Comment:
By default, an option takes a value. For boolean flag you'll need to either pass 2 values (suggested way):
```suggestion
    "--create_table/--no-create-table",
```
or pass `flag=True` (less-suggested way).
In both cases, it's always best to explicitely provide a default value (default default is "False")

Reply:
oh thank you I missed that, I'll just changed a bit so naming is consistent

```suggestion
    "--create-table/--no-create-table",
```